1-org/envs/shared/scc_notification.tf

```
@@ -29,28 +29,14 @@ resource "google_pubsub_subscription" "scc_notification_subscription" {
   project = module.scc_notifications.project_id
 }
 
-module "scc_notification" {
-  source  = "terraform-google-modules/gcloud/google"
-  version = "~> 1.1.0"
 
-  additional_components = var.skip_gcloud_download ? [] : ["alpha"]
+resource "google_scc_notification_config" "scc_notification_config" {
+  config_id    = var.scc_notification_name
+  organization = var.org_id
+  description  = "SCC Notification for all active findings"
+  pubsub_topic =  google_pubsub_topic.scc_notification_topic.id
 
-  create_cmd_entrypoint = "gcloud"
-  create_cmd_body       = <<-EOF
-    scc notifications create ${var.scc_notification_name} --organization ${var.org_id} \
-    --description "SCC Notification for all active findings" \
-    --pubsub-topic projects/${module.scc_notifications.project_id}/topics/${google_pubsub_topic.scc_notification_topic.name} \
-    --filter "${var.scc_notification_filter}" \
-    --project "${module.scc_notifications.project_id}" \
-    --impersonate-service-account=${var.terraform_service_account}
-EOF
-
-  destroy_cmd_entrypoint = "gcloud"
-  destroy_cmd_body       = <<-EOF
-  scc notifications delete organizations/${var.org_id}/notificationConfigs/${var.scc_notification_name} \
-  --impersonate-service-account ${var.terraform_service_account} \
-  --project "${module.scc_notifications.project_id}" \
-  --quiet
-  EOF
-  skip_download          = var.skip_gcloud_download
+  streaming_config {
+    filter = "category = \"OPEN_FIREWALL\" AND state = \"ACTIVE\""  #var.scc_notification_filter
```

I think this should be
```suggestion
    filter = var.scc_notification_filter
```

---

Maybe the current *default* value for the  scc_notification_filter 

https://github.com/terraform-google-modules/terraform-example-foundation/blob/a9145f48fbf53df3fa9aaabc31b887b6e1a9f11d/1-org/envs/shared/variables.tf#L81-L86

is over escaped and the `\\\"` could be shortened to just `\"`

---

Oops, I forgot to replace the testing value and will update it in next commit